packages/authenticator/amplify_authenticator/lib/src/state/authenticator_state.dart

```diff
@@ -168,24 +167,25 @@ class AuthenticatorState extends ChangeNotifier {
   String _newPassword = '';
 
   /// The value for the country code portion of the phone number field
-  Country get country => _country;
+  DialCode get dialCode => _dialCode;
 
-  set country(Country newCountry) {
-    final oldCountry = _country;
+  set dialCode(DialCode newDialCode) {
+    final oldDialCode = _dialCode;
     final currentPhoneNumber =
         authAttributes[CognitoUserAttributeKey.phoneNumber];
     if (currentPhoneNumber != null) {
       authAttributes[CognitoUserAttributeKey.phoneNumber] =
           currentPhoneNumber.replaceFirst(
-        oldCountry.value,
-        newCountry.value,
+        oldDialCode.value,
+        newDialCode.value,
       );
     }
-    _country = newCountry;
+    _dialCode = newDialCode;
     notifyListeners();
   }
 
-  Country _country = initialCountryCode;
+  late DialCode _dialCode = defaultDialCode;
+  DialCode defaultDialCode;
```

Comment:
This value should never change once set.

```suggestion
  final DialCode defaultDialCode;
```